storage_service/locations/signals.py

```diff
@@ -44,4 +48,23 @@ def report_failed_fixity_check(sender, **kwargs):
 Full failure report (in JSON format):
 {}
 """.format(kwargs["uuid"], kwargs["location"], kwargs["report"])
+    package = models.Package.objects.get(uuid=kwargs['uuid'])
+
+    log = models.FixityLog()
+    log.package = package
+    report_data = json.loads(kwargs['report'])
+    log.error_details = report_data['message']
+    log.save()
+
     _notify_administrators(subject, message)
+
+
+@receiver(successful_fixity_check, dispatch_uid="fixity_check")
+def report_successful_fixity_check(sender, **kwargs):
+    package = models.Package.objects.get(uuid=kwargs['uuid'])
+
+    log = models.FixityLog()
+    log.package = package
+    report_data = json.loads(kwargs['report'])
+    log.success = True
+    log.save()
```

Comment:
Could this be merged with `report_failed_fixity_check` above? They seem very similar.  Also, report_data is not used in this function.